common/config/rush/pnpm-lock.yaml

```diff
@@ -734,6 +734,7 @@ packages:
   /@azure/ai-form-recognizer/3.1.0-beta.3:
     resolution: {integrity: sha512-+4QtFKNyxAmdqpcYjuAtmWKm/MuOe9kZsbpS9jA9h0YHzngNj5gc67AA4egV9BXOq9x+1phjYTNC/rxiOUr1uQ==}
     engines: {node: '>=8.0.0'}
+    deprecated: Please migrate to a supported (non-beta) version of this package
```

Comment:
who is pinning FR beta?

Reply:
It is the perf lib: https://github.com/Azure/azure-sdk-for-js/blob/ea8f74a97839a52863549e0931e2993e45de31d5/common/config/rush/pnpm-lock.yaml#L18293. I think it shouldn't. /cc @harsha-nalluru

Reply:
sdk/formrecognizer/perf-tests/ai-form-recognizer/package.json:12:    "@azure/ai-form-recognizer": "3.1.0-beta.3",

it could be that we forgot to upgrade when GA'ing the library

Reply:
This will be addressed separated